CONTRIBUTING.md

```diff
@@ -85,13 +85,13 @@ We are using [Vagrant](https://www.vagrantup.com/) to create our development env
 6. Start your development backend with:
 
     ```
-    vagrant exec RAILS_ENV=development ./script/start_test_backend
+    vagrant exec contrib/load_dev_backend.sh 
     ```
 
 7. Start your development OBS frontend:
 
     ```
-    vagrant exec rails s
+    vagrant exec 'cd src/api/ && rails s'
```

Comment:
With vagrant exec you can add options to specific commands. Like

fe.exec.commands %w(rails rake rspec bundle), directory: '/vagrant/src/api'